neuralmonkey/nn/utils.py

```diff
@@ -15,12 +15,8 @@ def dropout(variable: tf.Tensor,
         train_mode: A bool Tensor specifying whether to dropout or not
     """
     # Maintain clean graph - no dropout op when there is none applied
-    # TODO maybe use math.isclose instead of this comparison
-    with tf.variable_scope("dropout"):
```

Comment:
To by šlo, ale nerval bych to jako parametr, ale nastavil bych mu napevno "dropout"